pywhat/what.py

```diff
@@ -141,8 +169,14 @@ def main(text_input, rarity, include_tags, exclude_tags, only_text, key, reverse
         * what 'this/is/a/path'
 
     """
-
-    what_obj = What_Object(parse_options(rarity, include_tags, exclude_tags))
+    dist = Distribution(create_filter(rarity, include, exclude))
+    if disable_boundaryless:
+        boundaryless = Filter({"Tags": []})

```

Comment:
Why are we setting this to nothing? Can we have a comment explaining the rational please? :)